eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -177,10 +177,17 @@ private static void run(final List<String> opts) throws Exception {
         final String path = Arrays.stream(opts.get(0).split("\\."))
             .map(p -> String.format("EO%s", p))
             .collect(Collectors.joining("."));
-        final Phi app = Phi.class.cast(
-            Class.forName(path).getConstructor(Phi.class)
+        final Phi app;
+        try {
+            app = Phi.class.cast(
+                Class.forName(path).getConstructor(Phi.class)
                 .newInstance(Phi.Φ)
-        );
+            );
+        } catch (final ClassNotFoundException ex) {
+            throw new ExUnset(
+                String.format("Can not find \"%s\" object", opts.get(0))
```

Comment:
@levBagryansky what do you think if we write message using single quotes? 
```java
String.format("Can not find '%s' object", opts.get(0))
```
In that case we don't need to escape "

Reply:
@volodya-lombrozo ok